### ISSUE TITLE
Update Parts =>Sourcing documentation

### ISSUE THREAD

**Issue by dsvensson** [universam1/iSpindel#73]
This url has been out of stock for a long time now, and they don't reply to mail, at least not english mail it seems, http://cachers-world.de/de/Petling-XL

Perhaps you could throw up links to some other blessed containers?

**Comment by universam1** [comment]
This one here and he offers fitting sleds as well
https://3d-mechatronics.de/de/petling-3dm1-plastik-mit-deckel_239.html

**Comment by dsvensson** [comment]
And it's this one that fits or is the name misleading?

https://github.com/universam1/iSpindel/blob/master/drawer/iSpindel%20mechatronics.stl

**Comment by universam1** [comment]
exactly, and you can order it there as well

**Comment by bschuhma** [comment]
Has anyone figured out how to get one shipped to the US?  3d-mechatronics.de only ships to Europe it seems. :-(

Thanks,

Bret


On 8/16/17, 5:33 AM, "universam1" <notifications@github.com<mailto:notifications@github.com>> wrote:


exactly, and you can order it there as well

**Comment by Philusha1983** [comment]
you can find in this store on eBay:
https://www.ebay.com/sch/uniquegeocaches/m.html?_nkw=&_armrs=1&_ipg=&_from=

5 pcs XXL set of BIG PET Micro Geocaching container
 you can buy then as single aswell.
the size is similar but you will need to do some sand paper work on the 3d
printed part.
you can order the XL version but they smaller and you will need to do
changes in the 3d printed part to fit the smaller part.


Philip   ;)

**Comment by bschuhma** [comment]
Thanks, Philip.  I was actually hoping to get all the parts like the 47 Euro DIY kit at 3d-mechatronics.de so I don’t have to source each individually and/or MacGuyver it fit.

Regards,

Bret

On 8/16/17, 1:50 PM, "Philip Levin Slesarev" <notifications@github.com<mailto:notifications@github.com>> wrote:

you can find in this store on eBay:
https://www.ebay.com/sch/uniquegeocaches/m.html?_nkw=&_armrs=1&_ipg=&_from=

5 pcs XXL set of BIG PET Micro Geocaching container
you can buy then as single aswell.
the size is similar but you will need to do some sand paper work on the 3d
printed part.
you can order the XL version but they smaller and you will need to do
changes in the 3d printed part to fit the smaller part.


Philip ;)

**Comment by Philusha1983** [comment]
I can send you parts list if you want
 

*Philip* :)

**Comment by bschuhma** [comment]
Thanks for the kind offer, Philip.  I believe I’ve seen the parts list at the project site on github, but going to 3-5 different sources and paying shipping at each will about double my costs.  Maybe I’ll approach the people at 3d-mechatronics.de and see if they’re amenable.

Thanks again! :-)

Regards,

Bret

On 8/16/17, 1:59 PM, "Philip Levin Slesarev" <notifications@github.com<mailto:notifications@github.com>> wrote:

I can send you parts list if you want


*Philip* :)

**Comment by Philusha1983** [comment]
I ordered most of the parts from aliexpres and it all were less than 45$ 
per piece

*Philip* :)

**Comment by bschuhma** [comment]
Did you send the sled info to a 3D online printing service?  I checked about 6 months ago and I seem to recall it was $27+shipping.  AliExpress isn’t bad unless you’re in a hurry! ;-)  Cheap parts, but sloooooowwww  ;-).

Thanks,

Bret

On 8/16/17, 2:05 PM, "Philip Levin Slesarev" <notifications@github.com<mailto:notifications@github.com>> wrote:

I ordered most of the parts from aliexpres and it all were less than 45$
per piece

*Philip* :)

**Comment by bschuhma** [comment]
I’ll take another look at the parts and sourcing.  If I can do it for under $70USD it’ll be worth it – that’s what the kit would cost if I lived in Germany.  I’m sure my shipping will be higher, though, so maybe my breakeven point is closer to $90USD.

Regards,

Bret

On 8/16/17, 2:05 PM, "Philip Levin Slesarev" <notifications@github.com<mailto:notifications@github.com>> wrote:

I ordered most of the parts from aliexpres and it all were less than 45$
per piece

*Philip* :)
